legacy-fabric-entity-api-v1/common/src/main/java/net/legacyfabric/fabric/api/entity/EntityTypeIds.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2020 - 2024 Legacy Fabric
+ * Copyright (c) 2016 - 2022 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.legacyfabric.fabric.api.entity;
+
+import net.legacyfabric.fabric.api.util.BeforeMC;
+import net.legacyfabric.fabric.api.util.Identifier;
+import net.legacyfabric.fabric.api.util.SinceMC;
+
+public class EntityTypeIds {
+	public static final Identifier ITEM = id("item");
+	public static final Identifier XP_ORB = id("xp_orb");
+	@SinceMC("1.9")
+	public static final Identifier AREA_EFFECT_CLOUD = id("area_effect_cloud");
+	@SinceMC("1.11")
+	public static final Identifier ELDER_GUARDIAN = id("elder_guardian");
+	@SinceMC("1.11")
+	public static final Identifier WITHER_SKELETON = id("wither_skeleton");
+	@SinceMC("1.11")
+	public static final Identifier STRAY = id("stray");
+	@SinceMC("1.8.9")
+	public static final Identifier EGG = id("egg");
+	public static final Identifier LEASH_KNOT = id("leash_knot");
```

Comment:
unsure how far the compat goes, but leashes were added in 1.6.1

Reply:
We only go back to 1.7.10 for now, I'll add the annotation only when we go further back.